changelogs/unreleased/update-template-marker-openapi-docs.yml

```diff
@@ -0,0 +1,6 @@
+---
+description: 'Mark parameters in the path of an endpoint using `<param>` instead of `{param}` in the openapi documentation.'
+change-type: patch
+destination-branches: [master]
```

Comment:
Should we also merge this into iso8 and iso7 to keep the openapi documentation consistent across iso versions?

Reply:
Sounds like a good idea to me :+1: